sdk/src/feature_set.rs

```diff
@@ -760,6 +760,10 @@ pub mod deprecate_executable_meta_update_in_bpf_loader {
     solana_sdk::declare_id!("k6uR1J9VtKJnTukBV2Eo15BEy434MBg8bT6hHQgmU8v");
 }
 
+pub mod enable_zk_from_account {
```

Comment:
nit-up-to-you: maybe we can call it `enable_zk_proof_from_account` to be clearer?

Reply:
Okay, yes that will be clearer.